_templates/layout.html

```diff
@@ -0,0 +1,41 @@
+{%- extends "alabaster/layout.html" %}
```

Comment:
`alabaster` comes with its own `layout.html` template, so if somebody decides to change theme, it will be unclear what `layout.html` to compare with. But give me a moment..